### PR TITLE
refactor: cleanup `ExportStrongholdButton`

### LINKED ISSUE #7249: [Task]: Cleanup `ExportStrongholdButton`

### PR DESCRIPTION
**Pull request by evavirseda**
## Summary

Closes #7249 

## Changelog

```
- cleanup ExportStrongholdButton
```

### Platforms

> Please select any platforms where your changes have been tested.

-   **Desktop**
    -   [ ] MacOS
    -   [ ] Linux
    -   [ ] Windows
-   **Mobile**
    -   [ ] iOS
    -   [ ] Android

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to **test and verify** that your changes work as intended.

...

## Checklist

> Please tick the following boxes that are relevant to your changes.

-   [ ] I have followed the contribution guidelines for this project
-   [ ] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added or modified tests that prove my changes work as intended
-   [ ] I have verified that new and existing unit tests pass locally with my changes
-   [ ] I have verified that my latest changes pass CI workflows for testing and linting
-   [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/desktop/views/dashboard/settings/views/security/ExportStronghold.svelte [hunk 1]
```diff
@@ -2,8 +2,8 @@
     import { ExportStrongholdButton, Spinner, Text, TextType } from 'shared/components'
     import { localize } from '@core/i18n'
 
-    let exportBusy = false
-    let exportMessage = ''
+    let exportBusy: boolean = false
+    let exportMessage: string = ''
```
Comment (VmMad) [inline packages/desktop/views/dashboard/settings/views/security/ExportStronghold.svelte:6]:
```suggestion
    let isBusy: boolean = false
    let message: string = ''
```

File: packages/shared/components/buttons/ExportStrongholdButton.svelte [hunk 2]
```diff
@@ -5,12 +5,12 @@
...
     import { Button, ButtonSize } from 'shared/components'
 
-    export let isBusy = false
-    export let message = ''
-    export let showNotification = false
-    export let overrideTitle = ''
+    export let isBusy: boolean = false
+    export let message: string = ''
+    export let showNotification: boolean = false
+    export let overrideTitle: string = ''
```
Comment (VmMad) [inline packages/shared/components/buttons/ExportStrongholdButton.svelte:11]:
```suggestion
    export let title: string = ''
```

**Review by VmMad — CHANGES_REQUESTED** [review]
Props have weird names imo

**Review by cpl121 — APPROVED** [review]
NJ

**Review by begonaalvarezd — APPROVED** [review]

**Pull request merged**